omero/autogen_docs

```diff
@@ -12,7 +12,12 @@ WORKSPACE=${WORKSPACE:-$(pwd)}
 USER=${USER:-$(whoami)}
 
 echo "Copying history"
-cp $WORKSPACE/OMERO.server/history.rst omero/users/history.rst
+# inject link to change log
+touch $WORKSPACE/newfile.rst
```

Comment:
Is this needed?

Reply:
I think it is easier to follow what's happening starting with an empty file
i.e. 2 files are merged into one new file